src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx

```diff
@@ -130,12 +130,9 @@
     <value>A project for creating a command-line application that can run on .NET Core on Windows, Linux and MacOS.</value>
   </data>
   <data name="5" xml:space="preserve">
-    <value>XProj Files (*.xproj);*.xproj</value>
```

Comment:
@srivatsn, I removed the xproj resource in VSResources.resx and changed the other two numbers to be sequential with existing stuff. Is that going to break anything?